docs/rfcs/002-archival_storage/README.md

```diff
@@ -560,9 +584,33 @@ Response:
       }
     ]
   },
-  "archiveUrl" : "s3://archivebucket/born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476/" ,
-  "replicaUrl" : "https://archivebucket-replica.blob.core.windows.net/born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476/" ,
-  "accessUrl" : "s3://accessbucket/born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476/" ,
+  "archiveLocation": {
+    "type": "DigitalLocation",
+    "locationType": {
+      "type": "LocationType",
+      "id": "aws-s3-glacier",
+      "label": "AWS S3 - Glacier"
+    },
+    "url": "s3://bucketname.s3-eu-west-1.amazonaws.com/born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476"
+  },
+  "replicaLocation": {
```

Comment:
can these be lists? `replicaLocations=[{...}]`

Reply:
Updating as per our conversation.